cloudinary-image-management-and-manipulation-in-the-cloud-cdn/js/src/components/settings-page.js

```diff
@@ -79,7 +79,13 @@
 					field.removeClass( 'settings-valid-field' );
 					field.addClass( 'settings-invalid-field' );
 				}
-			}).trigger('change' );
+			}).trigger('change');
+
+			$('[name="cloudinary_sync_media[auto_sync]"]').change(function () {
+				if ($(this).val() === 'on') {
+					alert(cloudinaryJsData.auto_sync_notice)
```

Comment:
@dugajean 

Alerts works just fine, but WordPress has a modal system, the [Thickbox](https://codex.wordpress.org/Javascript_Reference/ThickBox) — that brings a more seamless experience. What do you think?

Reply:
Oh I wasn't aware. I'll use this 😉